docs/deploy-lightbend-cloudstate.adoc

```diff
@@ -59,58 +59,58 @@ $ akkasls version
+A Akka Serverless project is a space within your Akka Serverless cluster. In this space, you will deploy your Cloudstate-aware application. In this step we use the Akka Serverless CLI `akkasls` to:
 
-* Login to your Cloudstate cluster (which you got when signing-up for a <<signup-cloudstate-account,Cloudstate account>>)
-* Create a Cloudstate project
+* Login to your Akka Serverless cluster (which you got when signing-up for a <<signup-akka-serverless-account,Akka Serverless account>>)
+* Create a Akka Serverless project
 * Deploy your services in the project
 
-More details on `akkasls` is available at https://docs.lbcs.io/cli/index.html[akkasls reference documentation].
+More details on `akkasls` is available at https://docs.cloudstate.com/reference/cli-index.html[akkasls reference documentation].
 
 
 [source,shell]
 ----
-#===STEP1: connect akkasls to your Cloudstate account
+#===STEP1: connect akkasls to your Akka Serverless account
 $ akkasls auth login
 
-#===STEP2: create a new Cloudstate project
+#===STEP2: create a new Akka Serverless project
 # Syntax: akkasls project new "project-friendly-name" "Project description"
 # NOTE: project-friendly-name: no space, only lower case and dash
-$ akkasls project new chat-application "Chat room app built with Cloudstate services"
+$ akkasls project new chat-application "Chat room app built with Akka Serverless services"
 
 Created project chat-application.
 To make this the currently active project, run: 'akkasls config set project chat-application'
   NAME               DESCRIPTION                                    STATUS    ID
-  chat-application   Chat room app built with Cloudstate services   pending   3d864594-fefe-4708-82da-da842e4cdcde
+  chat-application   Chat room app built with Akka Serverless services   pending   3d864594-fefe-4708-82da-da842e4cdcde
 
-#===STEP3: set the new Cloudstate project as "current"
+#===STEP3: set the new Akka Serverless project as "current"
 $ akkasls config set project chat-application
 
 Project set to chat-application
 ----
 
-IMPORTANT:: A newly created project is in `pending` status, ie. you cannot yet deploy on your Cloudstate cluster.
-Once the project is activated by a Cloudstate administrator (within a short moment). Its status will change to `active`, at which point, you can begin to deploy your services. In case it takes long time for your project to get activated, you can https://docs.lbcs.io/getting-started/create-project.html#_project_status[notify Cloudstate admin]
+IMPORTANT:: A newly created project is in `pending` status, ie. you cannot yet deploy on your Akka Serverless cluster.
+Once the project is activated by a Akka Serverless administrator (within a short moment). Its status will change to `active`, at which point, you can begin to deploy your services. In case it takes long time for your project to get activated, you can https://docs.cloudstate.com/getting-started/projects.html#_project_status[notify Akka Serverless admin]
```

Comment:
```suggestion
Once the project is activated by an Akka Serverless administrator (within a short moment). Its status will change to `active`, at which point, you can begin to deploy your services. In case it takes long time for your project to get activated, you can https://docs.cloudstate.com/getting-started/projects.html#_project_status[notify Akka Serverless admin]
```